src/components/resource/_fields/custom-embargo/custom-embargo-fields.js

```diff
@@ -15,7 +15,7 @@ import styles from './custom-embargo-fields.css';
 class CustomEmbargoFields extends Component {
   static propTypes = {
     change: PropTypes.func.isRequired,
-    initialValue: PropTypes.object,
+    initial: PropTypes.object,
     showInputs: PropTypes.bool
   };
 
```

Comment:
could you add the non-requred prop to the defaultProps, please?

Reply:
👍 Good catch. Added.